src/hooks/useJurorSubscriptionFees.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react'
```

Comment:
i think that this file does not belong to this pr right?

Reply:
Oops yeah definitely